knowledge_graph/make_graph.py

```diff
@@ -345,30 +190,26 @@ def makeGraph(onto_path, edge_path, output_folder_path):
     # print(list(default_world.inconsistent_classes()))
 
     # Read in the triples data
-    df = pd.read_csv(edge_path)
+    ## DMARX - csv via make_network.outputEdges()
+    #          via node_network.result
+    # ... If we've already processed the ontology through the Network object,
+    # why do we need to reload it here?
+    # Can we move add_ontology_data_to_graph_nodes to network_class?
+    df_edges = pd.read_csv(edge_path)
 
-    G = nx.DiGraph()  # There should not be duplicate edges that go the same direction.
-    # If so, need to throw an error.
+    G = nx.DiGraph()
```

Comment:
we still need a check step that throws an error if duplicate edges exist that go the same direction? Just leave a comment for that as a to do if not doing it now.